wrappers/PhyML/package_phyml_3_1/tool_dependencies.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="phyml" version="3.1">
+		<install version="1.0">
+			<actions os="linux" architecture="x86_64">
+				<action type="download_by_url">http://www.atgc-montpellier.fr/download/binaries/phyml/PhyML-3.1.zip</action>
+				<action type="move_file" rename_to="phyml">
+					<source>PhyML-3.1_linux64</source>
+					<destination>$INSTALL_DIR</destination>
+				</action>
+				<action type="chmod">
```

Comment:
Any special environment settings should be avoided. Galaxy should take care of permissions and path setting, so try without it and see if it works.